src/components/hooks/__mocks__/useActivityStream.js

```diff
@@ -1,6 +1,6 @@
 export default function useActivityStream(ID, adapter) {
-  const [roomID] = Object.keys(adapter.datasource);
+  const roomIDs = Object.keys(adapter.datasource);
   const rooms = adapter.datasource;
 
-  return ID === roomID ? rooms[`${ID}-activities`] : [];
+  return roomIDs.includes(ID) ? rooms[`${ID}-activities`] : [];
```

Comment:
sorry this should be: 
```js
roomIDs.includes(`${ID}-activities`) ? rooms[`${ID}-activities`] : [];
```

Reply:
There's actually an [`in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) operator.
```suggestion
  return `${ID}-activities` in roomIDs ? rooms[`${ID}-activities`] : [];
```

Reply:
then you need to change it everywhere else in our hooks. but is it necessary?

Reply:
If we use `in` it should be: 
```js
`${ID}-activities` in adapter.datasource ?
```

Otherwise we can use the `roomIDs` keys array. What would you prefer?

Reply:
yes, `adapter.datasource` should be fine. I was just looking at the one line without context 😛